app/src/main/AndroidManifest.xml

```diff
@@ -25,6 +25,17 @@
 
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
+            <intent-filter android:autoVerify="true">
+                <action android:name="android.intent.action.VIEW" />
+
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+
+                <data android:scheme="https" />
+                <data android:host="template.softteco.com.deep.link" />
+                <data android:scheme="http" />
+                <data android:host="template.softteco.com.deep.link" />
```

Comment:
can be simplified:

 <data android:scheme="http" android:host="template.softteco.com.deep.link" />
 <data android:scheme="https" android:host="template.softteco.com.deep.link" />